src/xsl/openrosa2html5form.xsl

```diff
@@ -1220,7 +1256,7 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
                 <xsl:with-param name="nodeset_u" select="$nodeset_used"/>
             </xsl:call-template>
         </xsl:variable>
-        <xsl:variable name="binding" select="/h:html/h:head/xf:model/xf:bind[@nodeset=$nodeset_used] | /h:html/h:head/xf:model/xf:bind[@nodeset=$nodeset]" />
+        <xsl:variable name="binding" select="key('nodeset-bindings',$nodeset_used) | key('nodeset-bindings', $nodeset)" />
```

Comment:
While I have no reason to doubt the safety of using `key` here, because I've verified other usage of the `nodeset-bindings` key, I want to call out that could not find an existing fixture which produces a different value for `$nodeset_used` and `$nodeset` here. In other words, all tests pass when using only one or the other.

I *was* able produce a difference with a contrived change to an existing fixture, but it isn't clear to me what non-contrived scenario this is supposed to cover.

Reply:
@eyelidlessness: the contrived change was related to relative vs absolute paths